wings/market/binance_market.pyx

```diff
@@ -1065,6 +1065,7 @@ cdef class BinanceMarket(MarketBase):
         return cancel_result
 
     cdef c_cancel(self, str symbol, str order_id):
+        self.c_stop_tracking_order(order_id)
```

Comment:
Orders are removed from tracking in ```execute_cancel```, only if the API request to cancel succeed.

Reply:
That's not happening in the current version. Its still present in inflight orders after succesful cancellation on binance. Let's discuss.